source/plugin-settings/obox-storage-plugin.rst

```diff
@@ -137,3 +137,21 @@ There are no migrated POP3 UIDLs. Don't try to look them up in any situation. No
 
 By default doing changes to folders (e.g. creating or renaming) uploads changes immediately to object storage. If this setting is enabled, the upload happens sometimes later (within metacache_upload_interval).
 
+.. _plugin-obox-storage-settting_obox_allow_copy_between_users:
+
+``obox_allow_copy_between_users``
+---------------------------------
+
+.. versionadded:: v2.3.11
+
+If set to yes this setting enables sharing mail-objects in storage between
+users. If enabled it can reduce the storage usage, for example mails
+with multiple recipients, will be only stored once, this also affects expunged
```

Comment:
extra comma "recipients will be". And ", this" -> ". This"

Reply:
Although what does it mean that it affects expunged mails?.. I suppose lazy_expunge refcounting?

Reply:
Yes you are right i removed the part about expunging here.